openupgrade_scripts/scripts/mail/16.0.1.10/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,221 @@
+---Models in module 'mail'---
+obsolete model mail.channel.partner
+obsolete model mail.resend.cancel [transient]
+model fetchmail.server (moved from fetchmail)
```

Comment:
@marielejeune is there any specific reason why I don't see _models_renames for fetchmail model ? Thanks

Reply:
Ok I understood, the model has been 'moved' and not 'renamed'